### PR TITLE
Extend tap targets for Footer links 

### PR DESCRIPTION
**Pull request by PaulChinkwende**
The tap target of list items in the footer now span the width of the entire list item.

* [x] Fixes https://github.com/bbc/simorgh/issues/393
* [x] Tests added for new features

* [x] Test engineer approval

**Test Instructions**
1. Git checkout tap-targets-footer-links.
2. `npm run storybook`.
3. Navigate onto the footer story.
4. Touch targets should match the UX specification.

### CODE REVIEW

**Comment by greenc05** [comment]
@PaulChinkwende I haven't checked this out  but I believe you will also need to move the padding set on the li's inside the links.... https://github.com/bbc/simorgh/blob/2d138bf51dc1f83dcd0fc374288e21998e404b07/src/app/components/Footer/index.jsx#L37

File: src/app/components/Footer/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -48,6 +49,21 @@ exports[`Footer should render correctly 1`] = `
   margin: 0;
 }
 
+.c5 {
+  color: #FFFFFF;
+  font-weight: 700;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  display: block;
+  display: inline-block;
```
Comment (pjlee11) [inline src/app/components/Footer/__snapshots__/index.test.jsx.snap]: display shouldn't be here twice

Comment (jtart) [reply src/app/components/Footer/__snapshots__/index.test.jsx.snap]: It is an internal of styled-components' `extend` that this happened, only the latter would be applied though. However, this is a moot point. `extend` is [being deprecated](https://github.com/styled-components/styled-components/issues/1546) for `styled(Component)`, which provides a cleaner CSS and the same functionality for this use-case, we should use that.

Comment (jtart) [reply src/app/components/Footer/__snapshots__/index.test.jsx.snap]: It is an internal of styled-components' extend that this happened, only the latter would be applied though. However, this is a moot point. extend is being deprecated for styled(Component), which provides a cleaner CSS and the same functionality for this use-case, we should use that.

Comment (jtart) [reply src/app/components/Footer/__snapshots__/index.test.jsx.snap]: It is an internal of styled-components' extend that this happened, only the latter would be applied though. However, this is a moot point. extend is being deprecated for styled(Component), which provides a cleaner CSS and the same functionality for this use-case, we should use that.

Comment (PaulChinkwende) [reply src/app/components/Footer/__snapshots__/index.test.jsx.snap]: It is there twice because it is overwritten

Comment (pjlee11) [reply src/app/components/Footer/__snapshots__/index.test.jsx.snap]: I get that it's the output of how styled components handles inheritance, I just don't like it 🤐

**Review by jtart — CHANGES_REQUESTED** [review]

**Comment by PaulChinkwende** [comment]
@jtart This change has been made.

**Comment by PaulChinkwende** [comment]
@greenc05 Your right. This has been done.

**Review by pjlee11 — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]
👍 Looks good!

**Review by jtart — DISMISSED** [review]

**Comment by PaulChinkwende** [comment]
**Test Instructions**
When I tab onto the header anchor tag "BBC News" should have a #FFFFFF(white) bottom-border/underline.

**Review by sareh — DISMISSED** [review]
👍 This looks good to me. 
It would be good to sit down with one of the UX designers to see if they have any comments on the spacing/padding. Both on the Grid-supporting browser and a non-Grid supporting browser (with BrowserStack).

**Review by jtart — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]

**Review by jtart — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]
👍 Looks good!

**Review by jtart — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]

**Comment by jamesbrumpton** [comment]
LGTM
Alt Text

**Review by sareh — APPROVED** [review]

**Review by jtart — APPROVED** [review]

**Pull request merged**